### DISCUSSION TITLE
Dealing with Sub-collections

### DISCUSSION THREAD

**Discussion by brndn-pex** [AntlerVC/firetable#951] [Support / Q&A] [answered]
Hi all, could someone let me know if there is a way to deal with sub-collections?

**Comment by harinij** [comment, marked as answer]
Hi @brndn-pex you have option to either see an existing sub-collection as a top level table (just provide the collection path while creating a table) or you can create sub-collection as a field, which is documented [here](https://docs.rowy.io/field-types/sub-table).
